packages/ui-tabs/src/Tabs/__tests__/Tabs.test.tsx

```diff
@@ -448,18 +448,17 @@ describe('<Tabs />', async () => {
       const subject = await mount(<Example width="150px" />)
       const tabs = await TabsLocator.find()
 
-      expect(await tabs.find('[class$="-tabs__scrollOverlay"]')).to.exist()
-      expect(await tabs.find('[class$="-tabs__scrollSpacer"]')).to.exist()
+      expect(await tabs.find('[class$="-tabs__endScrollOverlay"]')).to.exist()
+
+      const tabPanels = await tabs.findAllTabPanels()
+      tabPanels[2].click()
```

Comment:
I'm not sure how to test the scrolling event so I choose to click  the middle tab